.github/workflows/deploy-testnet.yml

```diff
@@ -231,6 +231,10 @@ jobs:
         scp deploy/testnet/package-lock.json ${{secrets[matrix.user_key]}}@${{secrets[matrix.host_key]}}:/home/${{ secrets[matrix.user_key] }}/atleta/package-lock.json
         scp deploy/config.env ${{secrets[matrix.user_key]}}@${{secrets[matrix.host_key]}}:/home/${{ secrets[matrix.user_key] }}/atleta/config.env
         scp chainspecs/chain-spec.testnet.json ${{secrets[matrix.user_key]}}@${{secrets[matrix.host_key]}}:/home/${{ secrets[matrix.user_key] }}/atleta/chainspec.json
+        ssh -o StrictHostKeyChecking=no ${{secrets.VALIDATOR_16_USER}}@${{secrets.VALIDATOR_16_HOST}} "mkdir -p /home/${{ secrets.VALIDATOR_16_USER }}/atleta/backup_script"
+        ssh -o StrictHostKeyChecking=no ${{secrets.VALIDATOR_16_USER}}@${{secrets.VALIDATOR_16_HOST}} "echo ${{ secrets.GCP_S3_KEY }} > /home/${{ secrets.VALIDATOR_16_USER }}/atleta/backup_script/gcp_key.json"
+        sed "s|\${VALIDATOR_16_USER}|${{secrets.VALIDATOR_16_USER}}|g" deploy/utils/backup_script/run_backup.sh > deploy/utils/backup_script/run_backup_fill.sh        
```

Comment:
Can we use this username as a script argument?
Or, if backup script runs by the host user, then just use his own `$HOME` variable.
I'm strictly against templating bash scripts as it involves unnecessary steps and is error prone.

Reply:
The blockchain db is not located in `$HOME`. for the `root` user, the `$HOME` directory is `/root`.
And in our case, the directories and files of the blockchain db are located in `/home/root/atleta`, which is not `$HOME`.

I can easily remove this approach, but there are validator nodes on the network where the default user is not `root`, but for example `ubuntu`.  And there, accordingly, the directory for the blockchain db is` /home/ubuntu/atleta`. This will be useful if we suddenly want to make backups of non-16th node.

Reply:
I have tested this method locally. Its worked

Reply:
I decided to simplify and declared via the `$USER` system variable to avoid the hardcore. 
Perhaps in the future, when migrating the infrastructure to a separate repository, we will adjust other paths for blockchain db without binding to user names. 

It is possible to declare a variable in a file via source `./user.env` or explicitly write text to a file, such as `root`, and then read it, for example` VALIDATOR_USER=$(cat ./user.env)`. But this is also a bad approach, because if we add a task to crontab, it will not start, since crontab requires absolute paths. And we will not be able to get the absolute path of this working directory. You can create a file in some other arbitrary location, but this would be logically incorrect.

I suggest that we just stop at something now, since there will be no ideal solution here.